### PR TITLE
py-logcat-color: new port, version 0.6.0

### PR DESCRIPTION
**Pull request by yan12125**
#### Description

Upstream supports only Python 2.x for now

###### Type(s)
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.13.4 17E202
Xcode 9.2 9C40b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? (N/A)
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
Tests do not pass. See comments around test.run in Portfile
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by yan12125** [comment]
Should I squash all commits into a single one?

**Comment by mojca** [comment]
Looks good to me. (Yes, ideally the commits would be squashed.)

**Comment by yan12125** [comment]
Got it! Squashed.

File: python/logcat-color_select/files/none [hunk 1]
```diff
@@ -0,0 +1 @@
+-
```
Comment (pmetzger) [inline python/logcat-color_select/files/none]: What is this for?

File: python/py-logcat-color/Portfile [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+categories-append   devel
+license             Apache-2
+maintainers         {gmail.com:yan12125 @yan12125} openmaintainer
+description         A colorful logcat
+long_description    \
+    A colorful and highly configurable alternative to the standard "adb logcat" \
+    command from the Android SDK
+platforms           darwin
+
+python.versions     27
```
Comment (pmetzger) [inline python/py-logcat-color/Portfile]: Note that Python27 isn't going to be supported relatively soon. If you like this port enough, I'd spend the several minutes porting it to Python 3 and send the patches upstream.

File: python/logcat-color_select/files/none (hunk 1 above)
Comment (yan12125) [reply python/logcat-color_select/files/none]: Actullay I don't quite understand how the select port group works. This is copied from https://github.com/macports/macports-ports/blob/master/python/flake8_select/files/none

File: python/py-logcat-color/Portfile (hunk 2 above)
Comment (yan12125) [reply python/py-logcat-color/Portfile]: I submitted one in 2016, but seems upstream is not interested in it :/
https://github.com/marshall/logcat-color/pull/24

File: python/logcat-color_select/files/none (hunk 1 above)
Comment (pmetzger) [reply python/logcat-color_select/files/none]: @mojca do you know how this works?

File: python/py-logcat-color/Portfile (hunk 2 above)
Comment (pmetzger) [reply python/py-logcat-color/Portfile]: Well, when the time comes you'll have that available at least.

File: python/logcat-color_select/files/none (hunk 1 above)
Comment (yan12125) [reply python/logcat-color_select/files/none]: Seems `-` instructs MacPorts to remove the target symbolic link. In this case the link /opt/local/bin/logcat-color is removed when ```port select --set logcat-color none``` is invoked

https://github.com/macports/macports-base/blob/8dda587a7359d8aa71fea3ad9744ba7019dcd1f2/src/macports1.0/macports.tcl#L4371-L4376

Comment (mojca) [reply python/logcat-color_select/files/none]: Yes, this is needed in order to delete the symlink. I slightly wonder if this is not a slight overkill, but well ... it's up to maintainer.

**Comment by pmetzger** [comment]
LGTM. I'm going to merge this.

**Pull request merged**